delta-proto/src/main/java/io/cdap/delta/proto/DeltaConfig.java

```diff
@@ -223,11 +223,12 @@ public DeltaConfig updateConfig(ApplicationUpdateContext upgradeContext)
     throws Exception {
     List<Stage> upgradedStages = new ArrayList<>();
     // Upgrade all stages.
-    for (Stage stage : getStages()) {
+    for (Stage stage : this.getStages()) {
```

Comment:
nit: why do we need to add all those "this" ?

Reply:
removed `this`.